WalletWasabi/Legal/Assets/LegalDocuments.txt

```diff
@@ -1,21 +1,21 @@
-    Our goal is to minimize the unintentionally leaked financial private information to third parties and to the receiver itself. The protection of your privacy and the protection of your (personal) data is our highest priority, which also defines the essence of our services. 
-    One instance of the Wasabi Wallet CoinJoin coordinating server is operated by a legal entity zkSNACKs Ltd. thus, in order to legally provide our services to our clients, we have defined the framework and the rules that both parties must follow in order to protect each other.
+    The WASABI team is committed to be fully transparent to its users, from its source code to the legal aspects of its services. The purpose of Wasabi Wallet is to provide privacy for its users’ Bitcoin transactions. This means that with the use of the wallet we would like to enable the users to make Bitcoin payments in such a way that their spending history, wallet balances, etc. are not visible to the receiving party. This does not necessarily mean that the user stays anonymous, since the receiving party might have to identify the sending party.
+    Our goal is to minimize the unintentional leaking or disclosure of private financial information to third parties and to the receiver of any transaction resulting from the use of the wasabi wallet. The protection of your privacy and the protection of your (personal) data is our highest priority, which also defines the essence of our services.
+    One instance of the Wasabi Wallet CoinJoin coordinating server is operated by a legal entity, zkSNACKs Ltd. Thus, in order to legally provide our services to our clients, we have defined the framework and the rules that both parties must follow in order to protect each other.
 
-    The most important parts of the below document is summarized in the following points:
-     - The service is open source under the MIT license.
-     - The service is provided on a non-custodial basis, safekeeping of keys are the sole responsibility of the user.
-     - The user is solely responsible to act according to local laws and regulations.
-     - We do not store any personally identifiable information. Moreover our trustless software architecture prevents us from gathering this information in the first place.
-     - Transaction fee is only charged by the service provider for CoinJoin transactions.
-     - We only provide written support and NEVER ask for recovery words, passwords or similar security critical information.
+    The most important parts of the document below is summarized in the following points:
+     - The service is open-source under the MIT license.
```

Comment:
There is `open source` without the hyphen in `WalletWasabi/Legal/Assets/LegalDocuments.txt`

Reply:
Fixed. TY